src/tree-node/index.css

```diff
@@ -21,6 +21,10 @@
       }
     }
   }
+
+  &.focused {
```

Comment:
This is new and even if we wanted to add this, can we not use `:focus` selector?

Reply:
No because the focus is actually on the input still. 
See https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html

We use aria-activedescendent to specify current "focused" item.